lib/util/exec/common.spec.ts

```diff
@@ -200,7 +200,7 @@ describe('util/exec/common', () => {
       ).rejects.toMatchObject({
         cmd,
         signal: exitSignal,
-        message: 'Process signaled with "SIGTERM"',
+        message: `Command failed: ${cmd}\n`,
```

Comment:
Can we indicate it was preempted by SIGTERM?

Reply:
yeah, the execError has signals and codes if available. 
only the `error.message` is changed
see exitCode below
```
DEBUG: Failed to generate composer.lock (repository=ladzaretti/composer-testing, branch=renovate/monolog-monolog-3.x)
       "err": {
         "name": "ExecError",
         "cmd": "C:\\Windows\\system32\\cmd.exe /d /s /c \"composer update monolog/monolog --with-dependencies --ignore-platform-req='ext-*' --ignore-platform-req='lib-*' --no-ansi -
-no-interaction --no-scripts --no-autoloader --no-plugins\"",
         "stderr": "'composer' is not recognized as an internal or external command,\r\noperable program or batch file.\r\n",
         "stdout": "",
         "options": {
           "cwd": "C:/Users/GABRIE~1/AppData/Local/Temp/renovate/repos/github/ladzaretti/composer-testing",
           "encoding": "utf-8",
           "env": {
             "COMPOSER_CACHE_DIR": "C:/Users/GABRIE~1/AppData/Local/Temp/renovate/cache/others/composer",
             "COMPOSER_AUTH": "{\"github-oauth\":{\"github.com\":\"**redacted**\"}}",
             "PATH": "C:\\Users\\GABRIE~1\\AppData\\Local\\Temp\\yarn--1659602435390-0.17226112613140776;C:\\projects\\ladzaretti-renovate\\node_modules\\.bin;C:\\Users\\GabrielLadza
retti\\AppData\\Local\\Yarn\\Data\\link\\node_modules\\.bin;C:\\Users\\GabrielLadzaretti\\AppData\\Local\\Yarn\\bin;C:\\Program Files\\libexec\\lib\\node_modules\\npm\\bin\\node-gyp-
bin;C:\\Program Files\\lib\\node_modules\\npm\\bin\\node-gyp-bin;C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\node-gyp-bin;C:\\Users\\GabrielLadzaretti\\Downloads\\jdk-11\\bin;
C:\\Program Files (x86)\\Common Files\\Oracle\\Java\\javapath;C:\\Windows\\system32;C:\\Windows;C:\\Windows\\System32\\Wbem;C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\;C:\\Windo
ws\\System32\\OpenSSH\\;C:\\Program Files\\dotnet\\;C:\\apache-maven-3.8.4-bin\\apache-maven-3.8.4\\bin;C:\\Program Files\\vscode;C:\\Gradle\\gradle-7.3.3\\bin;C:\\Program Files\\Git
\\cmd;C:\\Program Files\\nodejs\\;C:\\Program Files\\Docker\\Docker\\resources\\bin;C:\\ProgramData\\DockerDesktop\\version-bin;C:\\Program Files\\Go\\bin;C:\\Users\\GabrielLadzarett
i\\AppData\\Local\\Microsoft\\WindowsApps;C:\\Program Files\\JetBrains\\IntelliJ IDEA 2021.3.1\\bin;;C:\\Program Files\\JetBrains\\GoLand 2021.3.2\\bin;;C:\\Program Files\\JetBrains\
\WebStorm 2021.3.1\\bin;;C:\\Users\\GabrielLadzaretti\\AppData\\Roaming\\npm;C:\\Users\\GabrielLadzaretti\\go\\bin"
           },
           "maxBuffer": 10485760,
           "timeout": 900000
         },
         "exitCode": 1,
         "message": "Command failed: composer update monolog/monolog --with-dependencies --ignore-platform-req='ext-*' --ignore-platform-req='lib-*' --no-ansi --no-interaction --no-s
cripts --no-autoloader --no-plugins\n'composer' is not recognized as an internal or external command,\r\noperable program or batch file.\r\n",
         "stack": "ExecError: Command failed: composer update monolog/monolog --with-dependencies --ignore-platform-req='ext-*' --ignore-platform-req='lib-*' --no-ansi --no-interacti
on --no-scripts --no-autoloader --no-plugins\n'composer' is not recognized as an internal or external command,\r\noperable program or batch file.\r\n\n    at ChildProcess.<anonymous>
 (C:\\projects\\ladzaretti-renovate\\lib\\util\\exec\\common.ts:94:16)\n    at ChildProcess.emit (node:events:402:35)\n    at ChildProcess.emit (node:domain:475:12)\n    at Process.C
hildProcess._handle.onexit (node:internal/child_process:290:12)"
       }

```

Reply:
![image](https://user-images.githubusercontent.com/97394622/182812922-d4ea4b90-4a0c-4492-9dbf-6238b4c51a76.png)

Reply:
but i guess its available in the logs only

Reply:
misunderstood first, i guess for signals we can have a slightly different message which includes the signal?

but regardless both signal and exit codes are available in the logs.
should we add the signal to the error message?

Reply:
I think it's valuable to indicate that it didn't fail due to the package manager but instead due to being interrupted. Even if it's `Command failed ${cmd}\nInterrupted by SIGTERM`

Reply:
We could maybe even abort this branch in future btw so that it can be retried on the next run

Reply:
Or retry it immediately if that's elegant to do

Reply:
message changed to `Command failed ${cmd}\nInterrupted by ${signal}`, i omitted the stderr from the message, as it not quite helpful in this context, so does stdout. 
but still, both available in the logs though